<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-O5SIUU5" ignored="0" name="Test" tests="3" failures="1" timestamp="10 Nov 2017 07:20:16 GMT" time="0.104" errors="0">
  <testcase name="FirstTest" time="0.019" classname="execute_fail_test_case.execute_fail_case"/>
  <testcase name="SecondTest" time="0.006" classname="execute_fail_test_case.execute_fail_case2"/>
  <testcase name="ThirdTest" time="0.009" classname="execute_fail_test_case.execute_fail_case3">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at execute_fail_test_case.execute_fail_case3.ThirdTest(execute_fail_case3.java:11)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- ThirdTest -->
</testsuite> <!-- Test -->
